src/basis/basis.js

```diff
@@ -309,16 +341,19 @@ export class Basis {
      * This guarantees the independence from the order in the elements array.
      * @return {String}
      */
-    getAsSortedString(inStandardRepresentation = true) {
+    getAsSortedString() {
```

Comment:
This is a potentially dangerous behavior: removing default function args ...

Reply:
It's not used.

Reply:
OK